test/integration/targets/user/tasks/test_create_user_home.yml

```
@@ -152,3 +152,67 @@
         name: withskeleton
         state: absent
         remove: yes
+
+- name: Create user home directory with skel that contains symlinks
+  tags: symlink_home
+  become: True
+  vars:
+    flag: '{{tempdir.path}}/root_flag.conf'
+  block:
+    - name: make tempdir for skel
+      tempfile: state=directory
+      register: tempdir
+
+    - name: create flag file
+      file: path={{flag}} owner=root state=touch
+
+    - name: copy skell to target
+      copy:
+        dest: '{{tempdir.path}}/skel'
+        src: files/skel
+      register: skel
+
+    - name: create the bad symlink
+      file:
+        src: '{{flag}}'
+        dest: '{{tempdir.path}}/skel/should_not_change_own'
+        state: link
+
+    - name: "Create user home directory with /dev/null as skeleton"
+      user:
+        name: withskeleton
+        state: present
+        skeleton: "{{tempdir.path}}/skel"
+        createhome: yes
```

```suggestion
        createhome: yes
        home: /home/missing/withskeleton
```

chown_homedir() runs if the parent dir is missing for a new user, or if the user already exists but doesn't have a home dir yet.